tests/components/launch_library/test_config_flow.py

```diff
@@ -0,0 +1,54 @@
+"""Test launch_library config flow."""
+
+from homeassistant import data_entry_flow
+from homeassistant.components.launch_library.const import DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER
+
+from tests.common import MockConfigEntry
+
+
+async def test_import(hass):
+    """Test entry will be imported."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_IMPORT}, data={}
+    )
+    assert result.get("type") == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+
+
+async def test_user(hass):
+    """Test we can start a config flow."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+
+    assert result.get("type") == data_entry_flow.RESULT_TYPE_FORM
+    assert result.get("step_id") == "user"
+
+
+async def test_user_confirm(hass):
+    """Test we can finish a config flow."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}, data={}
+    )
+
+    assert result.get("type") == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+    assert result.get("result").data == {}
```

Comment:
Combine these in a single flow, walking through the start until the end of the flow.